shift_detector/checks/SimpleCheck.py

```diff
@@ -0,0 +1,161 @@
+                    'df2' not in numerical_comparison[column_name]['available_in']:
+                print('Column', column_name, 'not available in df2')
+
+            elif 'df2' in numerical_comparison[column_name]['available_in'] and \
+                    'df1' not in numerical_comparison[column_name]['available_in']:
+                print('Column', column_name, 'not available in df1')
+            else:
+                for metric in metrics:
+                    if metric == 'available_in':
+                        continue
+
+                    diff = self.relative_metric_difference(column_name, metric)
+                    if diff is not None:
+                        if abs(diff) > self.metrics_thresholds_percentage[metric]:
+                            print('shift in column', column_name, '\t', metric, self.difference_to_string(diff))
+
+    def print_categorical_report(self):
+        categorical_comparison = self.data['numerical_comparison']
+        for column_name, attribute in categorical_comparison.items():
+            for attribute_name, attribute_values in attribute.items():
+
+                if 'df1' not in attribute_values:
+                    attribute_values['df1'] = 0
+
+                if 'df2' not in attribute_values:
+                    attribute_values['df2'] = 0
+
+                diff = attribute_values['df1'] - attribute_values['df2']
+                if diff > self.categorical_threshold:
+                    print('shift in column ', column_name, 'attribute ', attribute_name, ': ', diff)
+
+    def print_report(self):
+        self.print_numerical_report()
+        self.print_categorical_report()
+
+
+class SimpleCheck(Check):
+    @staticmethod
+    def report_class():
+        return SimpleCheckReport
+
+    @staticmethod
+    def name() -> str:
+        return 'SimpleCheck'
+
+    def run(self, store):
+        df1_numerical = store[ColumnType.numerical][0]
+        df2_numerical = store[ColumnType.numerical][1]
+        df1_categorical = store[ColumnType.categorical][0]
+        df2_categorical = store[ColumnType.categorical][1]
+
+        numerical_comparison = self.compare_numerical_columns(df1_numerical, df2_numerical)
+        categorical_comparison = self.compare_categorical_columns(df1_categorical, df2_categorical)
+        return {"categorical_comparison": categorical_comparison, 'numerical_comparison': numerical_comparison}
+
+    @staticmethod
+    def compare_numerical_columns(df1, df2):
+        numerical_comparison = dict()
+        empty_metrics_dict = {'mean': {}, 'median': {}, 'min': {}, 'max': {}, 'quartile_1': {}, 'quartile_3': {},
+                              'uniqueness': {}, 'distinctness': {}, 'completeness': {}, 'std': {}, 'available_in': {}}
+
+        for df_name, df in [('df1', df1), ('df2', df2)]:
+            for column in df.columns:
+                if df_name == 'df1':
+                    numerical_comparison[column] = deepcopy(empty_metrics_dict)
```

Comment:
Why is this the same as 2 lines below?

Reply:
because it applies only in those two cases:
IF its the first column 
ELSE (second column) AND not already copied in the first column